components/Forms/LogTempForm/LogTempForm.js

```diff
@@ -84,15 +85,23 @@ const StyledSelectField = styled.select`
 
 // FUNCTIONALITY
 export default function LogTemperatureForm() {
+  const router = useRouter();
+  const { deviceId } = router.query;
+  const [device, setDevice] = useState(null);
   const [submitMessage, setSubmitMessage] = useState(false);
   const [devices, setDevices] = useLocalStorageState("devices", {
     defaultValue: [],
   });
+  useEffect(() => {}, [deviceId]);
   const {
     register,
     handleSubmit,
     formState: { errors },
-  } = useForm();
+  } = useForm({
+    defaultValues: {
+      date: new Date().toISOString().substr(0, 10),
```

Comment:
I recommend you to read about "Date / toLocaleDateString" on mdn (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString). With this method you have the ability to use options and format the date as you want.

Reply:
thank you for your recommendation :)